lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -364,7 +365,9 @@ Subscription will be done only once within first request to backend server.
 
 sub wait_for_messages {
     my ($self) = @_;
-    $self->{already_waiting} //= $self->redis->subscribe([$self->whoami], $self->redis->curry::weak::on(message => $self->curry::weak::_on_message));
+    $self->{already_waiting} //=
+        $self->redis->subscribe(['myriad.' . $self->whoami, $self->whoami],
```

Comment:
Let's not add `myriad` hardcoded in proxy, it's an open source module. Pass it as a config or as a parameter

Reply:
We can pass it from websocket.conf.